ipfscluster.go

```diff
@@ -83,6 +84,8 @@ type IPFSConnector interface {
 	// RepoStat returns the current repository size and max limit as
 	// provided by "repo stat".
 	RepoStat(context.Context) (*api.IPFSRepoStat, error)
+	// RepoGC returns the status of the garbage collection "repo/gc"
```

Comment:
A call to `/repo/gc` performs a garbage collection run on the IPFS node, it isn't a read operation of the status of the garbage collection

Reply:
Thanks, @lanzafame . 

Is this fine: `RepoGC(context.Context) error` ?

Reply:
Unresolving as this needs some clarification.

From my understanding the function proto in this interface must be similar to L#86. Happy to update the function based on the input. Thanks.